libs/features/mas/src/mas-commerce-service.js

```diff
@@ -160,20 +170,20 @@ export class MasCommerceService extends HTMLElement {
 
     flushWcsCache() {
         /* c8 ignore next 3 */
-        this.flushWcsCache();
+        this.flushCache();
```

Comment:
this is more confusing. Since we manage wcs and fragments caches,  `this.flushCache` doesn't imply that it only flushes wcs cache.